build.fsx

```diff
@@ -47,21 +45,8 @@ let Run workingDirectory fileName args =
 
     ProcessResult.New code messages errors
 
-let GetHomeDirectory =
-    let result = Run currentDirectory "cmd" "/c \"echo %USERPROFILE%\""
-    result.Messages.[0]
-
-let GetDnvmHome =
-    let homeDirectory =
-        if buildServer = BuildServer.AppVeyor
-            then "C:\\Program Files\\Microsoft DNX\\Dnvm\\"
-            else (GetHomeDirectory + "\\.dnx\\bin\\")
-
-    homeDirectory
-
-let GetDnxHome =
-    let homeDirectory = GetHomeDirectory
-    homeDirectory + ("\\.dnx\\runtimes\\dnx-" + runtime + "-win-" + architecture + "." + runtimeVersion + "\\bin\\")
+let dotnetHome = "C:\\Program Files\\dotnet\\"
```

Comment:
I should have reviewed this more carefully. This won't work on machines where `dotnet` doesn't exist. Is there a command to download dotnet.exe? If not, we should put dotnet.exe in the `tools` directory.